WoWsShipBuilder.Web/Dialogs/ShipAndShellSelectionDialog.razor

```diff
@@ -142,7 +142,7 @@
                 artilleryData.Add(shellIndex, new());
             }
         }
-        ShellSelectionWrapper wrapper = new(shipIndex, artilleryData, -1);
+        ShellSelectionWrapper wrapper = new(shipIndex, artilleryData, -1, -1);
```

Comment:
Consider using a different data structure if values are impossible to know at this point. Otherwise, consider making them nullable to indicate that these values are not always present on initialisation

Reply:
This is still not solved